action.yml

```diff
@@ -2,6 +2,33 @@ name: 'Synopsys Unified'
 description: 'Scan using synopsys tools'
 author: 'Synopsys Inc'
 inputs:
+  coverity_url:
+    description: 'Coverity url'
+    required: true
```

Comment:
If we are making the required as true, in case of other than coverity flow, github may expect this input. 
Please verify, if other flow can without this input. I believe we can validate these input as part of input validation in case of coverity flow

Reply:
done